modules/ROOT/pages/to-manage-categories.adoc

```diff
@@ -4,16 +4,16 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange Administrators and Organization Owners can create categories to group Exchange access types.
-Categories let users organize assets into different groups to improve asset browsing and discovery.
+Use categories to organize Exchange assets into groups and improve asset browsing and discovery.
 
-For example, a category called API Type with possible values `System API`, `Experience API`,
-and `Integration API` is applied to assets of type REST API, SOAP API, and HTTP API:
+For example, a category called *API Type* with possible values *System API*, *Experience API*,
+and *Integration API* is applied to assets of type *REST API*, *SOAP API*, and *HTTP API*:
```

Comment:
Are these bold because they are displayed as labels in the image?
If not, unbold them.

Reply:
Same for the final line below.

Reply:
Per our Slack discussion, these are not labels, so I removed the bold formatting.